Makefile

```diff
@@ -58,7 +58,7 @@ gomoddownload:
 
 .PHONY: gotest
 gotest:
-	@$(MAKE) for-all-target TARGET="test test-unstable"
+	@$(MAKE) for-all-target TARGET="test"
```

Comment:
Why is this removed? What if we have other unstable features to test?

Reply:
I think we should go with feature gates for unstable features from now on, this is an old unstable feature before the gates.